src/TrackPanel.cpp

```diff
@@ -1368,6 +1368,20 @@ struct ChannelGroup final : TrackPanelGroup {
 
       return { Axis::Y, std::move( refinement ) };
    }
+
+   void Draw(TrackPanelDrawingContext& context, const wxRect& rect, unsigned iPass) override
+   {
+      if (iPass == TrackArtist::PassFocus && mpTrack->IsSelected())
+      {
+         auto trackRect = wxRect(
+            mLeftOffset, 
+            rect.y, 
+            rect.GetRight() - mLeftOffset, 
+            rect.height);
+         TrackArt::DrawCursor(context, trackRect, mpTrack.get());
+      }
+   }
+
```

Comment:
I do not approve this commit.

Was this change of appearance explicitly specified?  To draw the zero-width selection over the affordance too?

As implemented here, the line will also be stroked

Reply:
I've asked and got positive answers
https://www.figma.com/file/odPgVM0lWc6MdsJWxpgPJl/Audacity?node-id=1998%3A42241

Reply:
I'm not sure which picture I should look at there.

Still, if it is the correct design, the implementation should use Overlay classes.

Reply:
When I click that link, the picture shows me a mono track, with a non-point selection -- and it id NOT painting over the affordance.  And that behavior will happen with no other change.

Where is there a picture specifying a change for drawing of point selection?  And also showing a stereo example?